cmake/winml_unittests.cmake

```diff
@@ -43,14 +43,36 @@ function(add_winml_test)
   if (_UT_DEPENDS)
     add_dependencies(${_UT_TARGET} ${_UT_DEPENDS})
   endif()
-  target_link_libraries(${_UT_TARGET} PRIVATE ${_UT_LIBS} gtest ${onnxruntime_EXTERNAL_LIBRARIES} winml_lib_common onnxruntime)
+  target_link_libraries(${_UT_TARGET} PRIVATE ${_UT_LIBS} gtest winml_google_test_lib ${onnxruntime_EXTERNAL_LIBRARIES} winml_lib_common onnxruntime)
 
   add_test(NAME ${_UT_TARGET}
     COMMAND ${_UT_TARGET}
     WORKING_DIRECTORY $<TARGET_FILE_DIR:${_UT_TARGET}>
   )
 endfunction()
 
+function(get_winml_test_scenario_src
+  lotus_root_dir
+  output_winml_test_scenario_src
+)
+  set(WINML_TEST_SRC_DIR ${lotus_root_dir}/winml/test)
```

Comment:
>WINML_TEST_SRC_DIR [](start = 6, length = 18)

can we use this from the outer scope? it looks like it is references in other functions?